src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[]args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
 
+        bulldozer.doWork();
+        excavator.doWork();
+        truck.doWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
+        truck.stopWork();
```

Comment:
let's create an array with machines and use for loop to call methods on all these objects